src/applications/edu-benefits/components/EducationWizard.jsx

```diff
@@ -390,22 +390,22 @@ export default class EducationWizard extends React.Component {
                           science, technology, engineering, or math (STEM),{' '}
                           <b>or</b>
                         </li>{' '}
-                        <li>
-                          {' '}
+                        <li className="li-styling">
                           You've already earned a STEM bachelor’s degree and are
-                          pursuing a teaching certification.{' '}
+                          working toward a teaching certification, <b>or</b>
+                        </li>{' '}
+                        <li className="li-styling">
+                          {' '}
+                          You've already earned a STEM bachelor's or graduate
+                          degree and are pursuing a covered clinical training
+                          program for health care professionals. <br />
                           <a
+                            aria-label="See eligible degree programs, opening in new tab"
```

Comment:
Shouldn't the aria-label include "and clinical training"?

Reply:
Fixed merged in from master